xbmc/platform/Platform.h

```diff
@@ -34,4 +34,12 @@ class CPlatform
    * or initialisation (like setting environment variables for example)
    */
   virtual bool Init();
+
+  /**\brief Enables user interacted startup activation and configuration of disabled add-ons
+   */
+  void EnableConfigureAddonsAtStartup() { m_configureAddonsAtStartup = true; };
+  bool IsConfigureAddonsAtStartupEnabled() const;
```

Comment:
Thinking about this one more time, do we actually need two members functions and a new member?

I think defining 

```c++
virtual bool IsConfigureAddonsAtStartupEnabled() { return false; }
```

 here and ovverriding it 
 
 ```c++
bool IsConfigureAddonsAtStartupEnabled() override { return true; }
```

in xbmc/platform/linux/PlatformLinux.cpp  does the job. This also makes the new member `m_configureAddonsAtStartup` obsolete and also the call to `EnableConfigureAddonsAtStartup();` in xbmc/platform/linux/PlatformLinux.cpp.

Reply:
I agree with this. 😊

Reply:
sounds absolutely reasonable

Reply:
needs to be overridden in `PlatformPosix` as it's the base for `PlatformLinux`; virtual/override not necessary i think, but doesn't harm though?

Reply:
> virtual/override not necessary 

`virtual` is absolutely necessary to make this work.
`override` could be omitted, but most probably produces a compiler warning. IIRC, out coding guidelines say that we have to use `override`